crm_phonecall/models/crm_phonecall.py

```diff
@@ -69,6 +69,8 @@ class CrmPhonecall(models.Model):
     date_closed = fields.Datetime(string="Closed", readonly=True)
     date = fields.Datetime(default=lambda self: fields.Datetime.now())
     opportunity_id = fields.Many2one(comodel_name="crm.lead", string="Lead/Opportunity")
+    direction = fields.Selection([("in", "In"), ("out", "Out")])
+    from_partner_id = fields.Many2one("res.partner", "From Contact")
```

Comment:
I thought this was in `crm_phonecall` already.

Now that I think of it, I believe we got it wrong.
There's probably no need for this field, and we're probably misusing the `partner_id` field.

IMO:

- `partner_id` should always be the external partner
- `from_partner_id` shouldn't be required
- `user_id` should be the internal user

It also makes sense because the Phonecall smart button in the res.partner form shows calls filterting by `partner_id` field, not `from_partner_id`.

I think we can keep it like this in our module, because we already consumed a more than the budget, we we shouldn't push it to the OCA as it doesn't make sense for them